bindings/wasm/bindings.js

```diff
@@ -415,36 +415,36 @@ Module.setup = function() {
   Module.ManifoldError = function ManifoldError(code, ...args) {
     let message = 'Unknown error';
     switch (code) {
-      case Module.status.NonFiniteVertex.value:
+      case 'NonFiniteVertex':
```

Comment:
Could we get rid of this whole switch statement if we used the descriptor strings as the status strings directly? e.g. use "Non-finite vertex" instead of "NonFiniteVertex"?

Reply:
Yes, but not sure if it is the best approach. I feel like the status name should be separated from the actual error message, and while we are using strings, what we want is just the status name.

Reply:
To me the only separation between status name and message is whether it contains spaces. C++ is annoying because enums can't have spaces and can't be coerced into strings, so you end up needing a bunch of duplication. If we're just using strings here anyway, seems like we can short circuit that. But you're welcome to merge this if you disagree - it's not a big deal.

Reply:
I think keeping closer to the C++ API is the right thing to do here, it minimizes surprises.